app/src/app/components/contents/admin/AdminLayout.tsx

```diff
@@ -0,0 +1,31 @@
+import { SidebarItem } from "../../../../lib/types/SidebarItem";
+import { Sidebar } from "../common/Sidebar";
+import { AdminOutlet } from "./AdminOutlet";
+import { useUser } from "../../providers/UserProvider";
+import { hasGlobalPacketManagePermission, hasUserManagePermission } from "../../../../lib/auth/hasPermission";
+
+export const AdminLayout = () => {
+  const { authorities } = useUser();
+  const sidebarItems: SidebarItem[] = [];
+  if (hasUserManagePermission(authorities)) {
+    sidebarItems.push(
+      {
+        to: "/manage-roles",
+        title: "Manage Roles"
+      },
+      {
+        to: "/manage-users",
+        title: "Manage Users"
+      }
+    );
+  }
+
+  if (hasGlobalPacketManagePermission(authorities)) {
```

Comment:
```suggestion
  if (canManageAllPackets(authorities)) {
```

Reply:
Does this need bumping? Already mentioned in the below comment albeit I decided to leave it as "hasGlobalPacketManagePermission" rather than "hasGlobalPacketManage" to retain consistency with "hasUserManagePermission" in the same file. Could slice "Permission" off both if you prefer the shorter version?